doc/source/whatsnew/v0.25.0.rst

```diff
@@ -124,7 +124,7 @@ Bug Fixes
 ~~~~~~~~~
 - Bug in :func:`to_datetime` which would raise an (incorrect) ``ValueError`` when called with a date far into the future and the ``format`` argument specified instead of raising ``OutOfBoundsDatetime`` (:issue:`23830`)
 - Bug in an error message in :meth:`DataFrame.plot`. Improved the error message if non-numerics are passed to :meth:`DataFrame.plot` (:issue:`25481`)
--
+- Segmentation fault when construct :class:`DataFrame` from non-empty tuples (:issue:`25691`)
```

Comment:
Bug in

Reply:
move this to Bug Fixes reshaping section

Reply:
Done.